tests/data_context/test_data_context_variables.py

```diff
@@ -505,3 +531,69 @@ def test_file_data_context_variables_e2e(
         file_data_context.variables.plugins_directory == value_associated_with_env_var
     )
     assert config_saved_to_disk.plugins_directory == f"${env_var_name}"
+
+
+@pytest.mark.integration
+@pytest.mark.cloud
```

Comment:
Do we care about this mark? Currently used in the `dgtest-cloud-overrides` section of the `dependency_graph` pipeline to ensure we always test our Cloud integrations - if we want to keep this, let's be consistent and ensure we mark other tests the same way.